.versions.yml

```diff
@@ -2,7 +2,7 @@ files:
   - path: Cargo.toml
     pattern: "\\[package\\][\\s\\S]*?version = \"{version}\""
   - path: README.md
-    pattern: "    image: public.ecr.aws/prima/localauth0:{version}"
+    pattern: "    image: public.ecr.aws/c6i9l4r6/localauth0:{version}"
```

Comment:
Why the spaces in the pattern?

Reply:
Would it work without them?

Reply:
No idea, that's just how it was before. Removed them